sam.local.yml

```diff
@@ -0,0 +1,23 @@
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: AWS::Serverless-2016-10-31
+Description: 'recap-hold-request-consumer'
+
+Resources:
+  RecapHoldRequestConsumer:
+    Type: AWS::Serverless::Function
+    Properties:
+      Handler: main.handle_event
+      Runtime: ruby2.5
+      Timeout: 30
```

Comment:
Does this require an `Events` field?

Reply:
Good question. I tend to add an `Events` definition for services because that allows us to run `sam local start-api` to run a local server. In that case, the `Events` definition is necessary to establish what routes to expose on the local server. For consumer/listener apps, there's not really an equivalent benefit. For running local ad hoc tests I tend to just run `sam local invoke` and give it the event file. No `Events` definition is required to do that. The events file itself contains all information about what kind of event it is as well as what the payload is.